scipy/signal/_peak_finding.py

```diff
@@ -420,7 +1000,7 @@ def filt_func(line):
 def find_peaks_cwt(vector, widths, wavelet=None, max_distances=None,
```

Comment:
I find this function (``find_peaks_cwt``) particularly confusing. It mixes the signal pre-processing and the find peaks function.

Maybe it is worth including some kind of note mentioning that the (new) function ``find_peaks`` provide an easier, matlab-like, interface. This would probably make the life of someone looking for it easier.

In the long term, I think we should even considering deprecating this function.